scripts/projectUpdater.ps1

```diff
@@ -61,6 +61,14 @@ function _openMergeWindow ($_updatedFile, $_currentFile) {
     }
 }
 
+# Don't allow the update of the project if there is no git repository initiated, to avoid losing track of the changes applied in the update
```

Comment:
I do not know about this one, perhaps instead of forcing we could show a prompt for the user to accept continuing running

Reply:
Sure, fixed